### PR TITLE
Fix: Client feedback fixes for subscription cancellation

### PR DESCRIPTION
**Pull request by deevannaveed**
## Context
The purpose of this PR is to make a few improvements based on client feedback.
1. Make the text size identical for "Payment history" and "Expires on" and aligned it to the left in same level.
2. Added a badge to make the inactive and expiry date more obvious. 
3. Aligned the text on the left in same level.
4. Added a new filed in the invoices table "Valid until" which indicates the end date for that invoice. 
5. Fetch invoices in descending order so that the users can see the latest ones first. 

Note: I am having a issue with the "Valid until" date, its showing the same date, need to figure it out.

## Test
From the deploy preview of this PR make sure that all the 5 points are satisfied.

### CODE REVIEW

File: resources/badge/index.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
+import React from "react";
+
+interface BadgeProps {
+  children: React.ReactNode;
+  size?: "small" | "medium" | "large";
+  variant?: "success" | "danger" | "warning" | "info" | "default";
+  corners?: "rounded" | "pill" | "sharp";
+}
+
+const Badge: React.FC<BadgeProps> = ({
```
Comment (abelbordonado) [inline resources/badge/index.tsx:10]: Nice new component

File: resources/tests/seed.ts [hunk 2]
```diff
@@ -30,6 +30,9 @@ export class Seed {
       return {
         invoice_id: faker.string.uuid(),
         created_at: faker.date.past().toISOString(),
+        metadata: {
+          period_end: faker.date.future().toISOString(),
+        },
```
Comment (abelbordonado) [inline resources/tests/seed.ts:35]: I think we are mimicking supabase structure, we can make a flat own structure and in the service map it correctly

File: modules/subscriptions/subscriptions/langs/en.json [hunk 3]
```diff
@@ -40,6 +40,7 @@
   "invoice-history": {
     "invoice-id-th": "Invoice Id",
     "date-th" : "Date",
+    "validity-th" : "Valid until",
     "amount-th" : "Amount",
     "action-th" : "Actions",
     "total-label" : "Total Records:",
```
Comment (kayiganwa04) [inline modules/subscriptions/subscriptions/langs/en.json:46]: is there any specific reason that we have : behind Total Records?

**Comment by kayiganwa04** [comment]
@deevannaveed  I have tested the deployment preview and everything is looking great as indicated in the PR description and yeah expect the date issue on date and valid until it shows the same date as the date when the user subscribed

File: modules/subscriptions/subscriptions/langs/en.json (hunk 3 above)
Comment (abelbordonado) [reply modules/subscriptions/subscriptions/langs/en.json:46]: `Total Records: 1 of 3`

**Pull request merged**